app/helpers/word-pluralize.js

```diff
@@ -0,0 +1,12 @@
+import {helper} from 'ember-helper';
+
+export function wordPluralize(params) {
```

Comment:
Oh nice! I spent a bit of time googling because I thought that would be the case but my google foo failed me in this case.